packages/api/src/schema.ts

```diff
@@ -1619,16 +1620,24 @@ const schema = gql`
     subscriptions: [Subscription!]!
   }
 
+  enum SubscriptionType {
+    RSS
+    NEWSLETTER
+  }
+
   type Subscription {
     id: ID!
     name: String!
-    newsletterEmail: String!
+    newsletterEmail: String
```

Comment:
I might need to update the clients for this change but shouldn't really be breaking since it would be set for all existing.